src/components/Apps/AccountCreateUpdateForm/const.js

```diff
@@ -121,11 +121,6 @@ export const accountFieldsMeta = (vm) => {
       el: {
         multiple: false,
         clearable: true,
-        disabled: {
-          get disabled() {
-            return vm.isDisabled
-          }
-        },
         ajax: {
           url: `/api/v1/accounts/accounts/su-from-accounts/?account=${vm.account?.id || ''}&asset=${vm.asset?.id || ''}`,
           transformOption: (item) => {
```

Comment:
The code you provided is incomplete since it's just a fragment without the rest of the file. However, here are some general points to consider:

* If `export {}` blocks were used instead of `{}`, then there would be no errors about missing names.

* The line that starts with an at symbol (`@`) seems to contain JavaScript comments instead of function syntax which might confuse anyone trying to understand its intent.

* It mentions Ajax calls using "transformOption", but in this context, I believe it likely means the `fetch` method with `axios`.

Since I don't know where else in the document or what specific details need clarification, please ensure all these steps are included before asking if it needs any additional support.